### ISSUE TITLE
Should provide a clear way to cache without shutting down the server?

### ISSUE THREAD

**Issue by ishanyang** [hapijs/vision#153]
Hello~
```
// express.js
app.use('/clearViewCache', function(req, res){
  pug.cache = {}
  return res.json({ok:1})
});
```
Do you think it is necessary to add a method in ' hapijs/vision/lib/index.js'?
```

server.decorate('server', 'clearViewCache', internals.clearViewCache);
```

**Comment by wswoodruff** [comment]
Hello @ishanyang! Are you using vision with Express?? Interesting -- Vision is a hapi plugin and uses hapi-specific methods, I don't think it'll work with Express.

Are you looking to programmatically clear the cache like #120 ? Or are you looking to clear the cache while working on your templates?

If you'd like to clear template cache during development there's a setting you can use -- in the plugin options when you register Vision, use `{ isCached: false }` and the files will be read each time they are requested. `isCached` defaults to true; during development you can turn it off

**Comment by wswoodruff** [comment]
Oh I'm sorry I see you were showing me the equivalent to this behavior in an express setup. Gotcha!

**Comment by wswoodruff** [comment]
@ishanyang Any updates on the issue?

**Comment by wswoodruff** [comment]
This will be fixed with #164

**Comment by wswoodruff** [comment]
I'm considering this as a duplicate of #120, closing and will fix in that PR ☝️
